ibm_watson/assistant_v1.py

```diff
@@ -2811,9 +3041,11 @@ def list_synonyms(self,
         if not value:
             raise ValueError('value must be provided')
         headers = {}
-        sdk_headers = get_sdk_headers(service_name=self.DEFAULT_SERVICE_NAME,
-                                      service_version='V1',
-                                      operation_id='list_synonyms')
+        sdk_headers = get_sdk_headers(
+            service_name=self.DEFAULT_SERVICE_NAME,
+            service_version='V1',
+            operation_id='list_synonyms',
+        )
```

Comment:
It looks like the vast majority of these changes are indent changes. Is this an intentional stylistic change or did we forget to run a style linter?

Reply:
This is due to changes to updates to the sdk generator